src/coreclr/System.Private.CoreLib/src/System/Reflection/Emit/AssemblyBuilder.cs

```diff
@@ -613,9 +535,7 @@ public void SetCustomAttribute(CustomAttributeBuilder customBuilder)
             }
         }
 
-        private void SetCustomAttributeNoLock(CustomAttributeBuilder customBuilder)
-        {
+        private void SetCustomAttributeNoLock(CustomAttributeBuilder customBuilder) =>
```

Comment:
It would be better to delete this method and inline its body in the one place that it is called. Same for the other SetCustomAttributeNoLock in this file.